config/webhook/webhookcainjection_patch.yaml

```
@@ -0,0 +1,3 @@
+- op: add
+  path: /metadata/annotations/service.beta.openshift.io~1inject-cabundle
+  value: "true" 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)